server.js

```diff
@@ -0,0 +1,29 @@
+const server = require('express')();
+
+const dbRouter = require('./data/db-router.js');
+
+server.use(require('express').json());
+
+// Logger example from Express Middleware TK:
+server.use(logger);
+
+// Possible problem with syntax here:
+server.use(require('cors')());
+
+server.use('/api/posts', dbRouter);
+
+server.get('/', (req, res) => {
+  res.send('Server running...');
+});
+
+module.exports = server;
+
+// Logger example from Express Middleware TK:
+function logger(req, res, next) {
+  console.log(
+    `[${new Date().toISOString()}] ${req.method} to ${req.url} from ${req.get(
+      'Origin'
```

Comment:
works but the req.get Origin portion is spat out in the console as `undefined`, this might be some extra data you need to instantiate along with the header? would take more research into what req.get is actually doing here